data/json/items/generic/string.json

```diff
@@ -107,5 +107,33 @@
     "volume": "1 L",
     "stack_size": 10,
     "weight": "80 g"
+  },
+  {
+    "id": "bowstring_makeshift",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "makeshift bowstring",
+    "symbol": ")",
+    "looks_like": "rope_6",
+    "material": [ "cotton" ],
+    "color": "light_gray",
+    "description": "A relatively strong cord, woven from whatever you had on hand. You could use this to string a simple bow.",
+    "volume": "349ml",
+    "stack_size": 3,
+    "weight": "80 g"
+  },
+  {
+    "id": "bowstring_advanced",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "bowstring",
+    "symbol": ")",
+    "looks_like": "bullwhip",
+    "material": [ "flesh", "dry_plant" ],
```

Comment:
This means it will always be made out of flesh, even if it's crafted entirely from plant matter.

Reply:
Is there a way to programmatically determine the material from the recipe's reagents? I was trying to follow DRY, and wasn't sure what the best path was here.

Reply:
Unfortunately, no, at the moment.

Reply:
Alright. It looks like the recipe for "long string" can take plant matter or sinew but always results in a cotton product - it seems reasonable to me to simply classify a bowstring as "dry plant" using that precedent, unless there's a "mixed fabric" or something else that would be more appropriate.

Reply:
I would probably make a judgement call on the "expected" ingredient and base it off that since we don't have a fancy system for it right now, which is i think how other items handle it. In this case that would "probably" be dry_plant, but for all i know people are collecting way more sinew than i think they are.

Reply:
Alright, I just went ahead and added a 'mixed fabric' material, applied it to the bowstrings. Not resolving this topic though, to give Kevin the opportunity to comment if he'd like.